packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/encounters-table/encounters-table.component.tsx

```diff
@@ -226,6 +230,10 @@ const EncountersTable: React.FC<EncountersTableProps> = ({
                     const isVisitNoteEncounter = (encounter: MappedEncounter) =>
                       encounter.encounterType === 'Visit Note' && !encounter.form;
 
+                    const isRfeCompatible = (encounter: MappedEncounter) =>
```

Comment:
```suggestion
                    const supportsEmbeddedFormView = (encounter: MappedEncounter) =>
```

Really nitpicky, but this might be clearer about its intention.